acrg/satellite/common.py

```diff
@@ -1312,6 +1342,97 @@ def name_pressure_file(filename,name='surface_pressure',column_names=["latitude"
     
     return ds
 
+
+def extract_files_oco2(directory, search_str=None, start=None, end=None, date_separator='', day=True):
```

Comment:
If this is necessarily specific to OCO-2 this may be better off in the `oco2.py` file rather than `common.py`